digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ViiteTierekisteriClient.scala

```diff
@@ -221,7 +221,10 @@ object ViiteTierekisteriClient {
     val response = client.execute(request)
     try {
       val statusCode = response.getStatusLine.getStatusCode
-      if (statusCode >= 500) {
+      if (statusCode == 500){
+        val errorMessage = parse(StreamInput(response.getEntity.getContent)).extractOpt[TRErrorResponse].getOrElse(TRErrorResponse("")) // would be nice if we didn't need case class for parsing of one attribute
+        ProjectChangeStatus(trProject.id, statusCode, errorMessage.error_message)
+      } else if (statusCode > 500) {
         logger.info(scala.io.Source.fromInputStream(response.getEntity.getContent).getLines().mkString("\n"))
         throw new RuntimeException("Unable to submit: Tierekisteri error 500")
```

Comment:
should we change message since in this case the error code is not 500?

Reply:
Changed text to " > 500"